react/ProductList.tsx

```diff
@@ -185,13 +154,22 @@ const ProductList: React.FC<Props> = (props) => {
           />
         </div>
       ) : null}
-      {unavailableGroups.map((group) => (
-        <ProductGroup
-          key={group.reduce((result, item) => `${result}#${item.id}`, '')}
-          {...props}
-          items={group}
-        />
-      ))}
+      {unavailableItems.map((item) => { 
+        const { userType, allowManualPrice, children } = props
+        const shouldAllowManualPrice = allowManualPrice && userType === CALL_CENTER_OPERATOR
+
+        return (
+          <ItemContextWrapper
+            key={`${item.uniqueId}-${item.sellingPrice}`}
```

Comment:
this is important

Reply:
(removing the `index` from the key, if compared to how the component was being inserted on the `ProductGroup` component)